test/e2e/self_provisioned_elasticsearch_kafka_test.go

```diff
@@ -23,11 +23,11 @@ import (
 	esv1 "github.com/jaegertracing/jaeger-operator/pkg/storage/elasticsearch/v1"
 )
 
-type SelfProvisionedTestSuite struct {
+type SelfProvisionedWithKafkaTestSuite struct {
```

Comment:
s/SelfProvisioned/SelfProvisionedES/? We can also self-provision Kafka, but I don't think that's what the test is about.

Reply:
Done.  It's a bit long but I never ever have to type SelfProvisionedESWithKafkaTestSuite again.  :-)